### PR TITLE
feat(rich-text-editor): toolbar menu by extension

### PR DESCRIPTION
**Pull request by zzzp9527**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/vue/src/rich-text-editor/src/components/toolbar-menu-item.tsx
Comment (zzzp9527) [inline packages/vue/src/rich-text-editor/src/components/toolbar-menu-item.tsx:1]: 重点文件

File: packages/renderless/src/rich-text-editor/vue.ts
Comment (zzzp9527) [inline packages/renderless/src/rich-text-editor/vue.ts:1]: 重点文件

File: packages/tiny-tiptap/src/components/toolbar/index.ts
Comment (zzzp9527) [inline packages/tiny-tiptap/src/components/toolbar/index.ts:1]: 重点文件

File: packages/tiny-tiptap/src/TiptapEditor.ts
Comment (zzzp9527) [inline packages/tiny-tiptap/src/TiptapEditor.ts:1]: 重点文件

**Review by fanbingbing16 — APPROVED** [review]

**Review by zzcr — APPROVED** [review]

**Pull request merged**